.travis.yml

```diff
@@ -19,6 +19,14 @@ before_install:
 script:
   - npm run check-ci
 
+  # Manually send coverage reports to coveralls.
+  # - Aggregate client results
+  # - Single server and func test results
+  # TODO(20): Server coverage
+  # TODO(21): Functional coverage
+  - ls coverage/client/*/lcov.info | cat
```

Comment:
I assume the pipe to `cat` is to force it into list form? Just in case it comes handy in the future, `ls -1` does that.